apps/src/lib/levelbuilder/ai-tutor/constants.ts

```diff
@@ -0,0 +1,48 @@
+const endpoints = [
+  {
+    id: 'ai-tutor',
+    name: 'AI Tutor + Webpurify',
+  },
+  {
+    id: 'llm-guard',
+    name: 'LLM Guard',
+  },
+];
+
+const genAIEndpoints = [
+  {
+    id: 'gen-ai-mistral-7b-inst-v01',
+    name: 'Mistral Base + Webpurify',
+  },
+  {
+    id: 'gen-ai-arithmo2-mistral-7b',
+    name: 'Mistral Arithmo + Webpurify',
+  },
+  {
+    id: 'gen-ai-biomistral-7b',
+    name: 'Mistral Biomistral + Webpurify',
+  },
+  {
+    id: 'gen-ai-karen-creative-mistral-7b',
+    name: 'Mistral Karen + Webpurify',
+  },
+  {
+    id: 'gen-ai-mistral-pirate-7b',
+    name: 'Mistral Pirate + Webpurify',
+  },
+];
```

Comment:
nit: I'd consider making this a little more "constant-y" (is that a word? 🙃), a la...

```
// Non-Gen AI endpoint constants
const AI_TUTOR_ENDPOINT = 'ai-tutor';
const LLM_GUARD_ENDPOINT = 'llm-guard';

// Gen AI endpoint constants
const GEN_AI_MISTRAL_BASE = 'gen-ai-mistral-7b-inst-v01';
const GEN_AI_ARITHMO = 'gen-ai-arithmo2-mistral-7b';
const GEN_AI_BIOMISTRAL = 'gen-ai-biomistral-7b';
const GEN_AI_KAREN_CREATIVE = 'gen-ai-karen-creative-mistral-7b';
const GEN_AI_PIRATE = 'gen-ai-mistral-pirate-7b';

// Endpoint Data
const endpoints = [
  {
    id: AI_TUTOR_ENDPOINT,
    name: 'AI Tutor + Webpurify',
  },
  {
    id: LLM_GUARD_ENDPOINT,
    name: 'LLM Guard',
  },
];

const genAIEndpoints = [
  {
    id: GEN_AI_MISTRAL_BASE,
    name: 'Mistral Base + Webpurify',
  },
  {
    id: GEN_AI_ARITHMO,
    name: 'Mistral Arithmo + Webpurify',
  },
  {
    id: GEN_AI_BIOMISTRAL,
    name: 'Mistral Biomistral + Webpurify',
  },
  {
    id: GEN_AI_KAREN_CREATIVE,
    name: 'Mistral Karen + Webpurify',
  },
  {
    id: GEN_AI_PIRATE,
    name: 'Mistral Pirate + Webpurify',
  },
];
```